Packs/Base/Scripts/SaneDocReportV2/SaneDocReportV2.py

```diff
@@ -12,16 +12,19 @@
 try:
     sane_json_b64 = demisto.args().get('sane_docx_report_base64', '').encode(
         'utf-8')
-    orientation = demisto.args().get('orientation', 'portrait').encode(
-        'utf-8')
-    paper_size = demisto.args().get('paperSize', 'A4').encode(
-        'utf-8')
+    orientation = demisto.args().get('orientation', 'portrait')
```

Comment:
why did you remove the encoding?

Reply:
I don't think there was a need, unless you think there are times it will be affected?

Reply:
tbh i'm not 100% sure, let's add this to be on the safe side

Reply:
Is this a convention in content? 
I remember that it works ok in pdf reports:
![image](https://user-images.githubusercontent.com/1269911/85228979-b1d05e80-b3ef-11ea-90d2-a021f01a8464.png)
(at least most of them). 
Maybe @glicht do you have an idea why we need to encode utf8?

Reply:
If the docker image is python3 we don't need it